backend/tracim_backend/radicale_plugins/radicale_none_right/__init__.py

```diff
@@ -0,0 +1,19 @@
+from radicale import pathutils
```

Comment:
If I understand right, `radicale_none_right` implements an access right policy which allows everybody.
If this is the case, wouldn't naming it `allow_all_access_rights` be more explicit?

Reply:
right